config_guide/cmdline/lib-include-config.md

```diff
@@ -15,15 +15,16 @@ These libraries have the suffix ".dll".
 The HELICS dll should look something like this "helicsSharedLib.dll".
 
 The ZMQ dll should look something like this
-"libzmq-v142-mt-gd-4_3_2.dll"
+"libzmq-v142-mt-gd-4_3_4.dll" (depending on your HELICS version, you may not have this)
 
 These could vary depending on your HELICS version and build.
 
 You will be copying these two libraries directly into the project like this:
-
-![](/config_guide/cmdline/demo-pictures/demo1.jpg)
-
-![](/config_guide/cmdline/demo-pictures/demo2.jpg)
+```
+cp <your path>/HELICS/build/bin/Debug/helicsd.dll .
+  
+cp <your path>/HELICS/build/bin/Debug/libzmq-v142-mt-gd-4_3_4.dll .
+```
```

Comment:
The exact library file names and paths they need to use will vary depending on the type of build -- if they do a release build in visual studio, it will be in the `bin/Release` subfolder instead of `bin/Debug`, and the dll file for helics will be `helics.dll` instead (not sure about the zmq dll). If they use a precompiled HELICS dll from GitHub it will be `helics.dll` in a `bin` folder (not located in a build subfolder), and headers will be in an includes subfolder.

Maybe an issue should be created for a future change to update the guide to be based off the installers on the GitHub releases for HELICS (or a build from source + install rather than using a HELICS build folder that theoretically should be possible to delete without breaking the omnet builds).